server/dex/dex.go

```diff
@@ -1463,6 +1469,20 @@ func (dm *DEX) EpochOrders(base, quote uint32) ([]order.Order, error) {
 	return dm.storage.EpochOrders(base, quote)
 }
 
+// Healthy returns the health status of the DEX.  This is true if
+// the storage does not report an error and the BTC backend is synced.
+func (dm *DEX) Healthy() bool {
+	if dm.storage.LastErr() != nil {
+		return false
+	}
+	if assetID, found := dex.BipSymbolID("btc"); found {
+		if synced, err := dm.assets[assetID].Backend.Synced(); err != nil || !synced {
+			return false
+		}
+	}
```

Comment:
We might as well check if other available asset backends are synced. If BTC is synced and other assets are not, then `DEX` is not `healthy`.

Reply:
Some assets and markets may be down but when others are running the status page shouldn't show the DEX being down.  BTC being down indicates disruption but DOGE does not.

Reply:
The logic behind not adding more elaborate information about market status is that the status platform really only needs to mark the DEX down if the DEX itself is unable to respond to client requests normally (due to network or hardware failure).  If the service is up then it can display the status of the individual markets.  The goal of this endpoint is to keep it simple in order to avoid false positives, the reason behind adding this one chain test is that the server really shouldn't ever fall behind the BTC chain tip - if it does, then that likely indicates some underlying problem.

Reply:
Gotcha, we’re just concerned about the HTTP API health here, yh?

Reply:
And a working database as well.